start_ocean.sh

```diff
@@ -138,17 +138,17 @@ while :; do
         # Version switches
         #################################################
         --latest)
-            export AQUARIUS_VERSION=${AQUARIUS_VERSION:-latest}
-            export BRIZO_VERSION=${BRIZO_VERSION:-latest}
-            export KEEPER_VERSION=${KEEPER_VERSION:-latest}
-            export PLEUSTON_VERSION=${PLEUSTON_VERSION:-latest}
+            export AQUARIUS_VERSION="latest"
```

Comment:
This changes the meaning of the `--latest` option (and the `--stable` option, below). The bash script code is easier to read, for sure, but the README.md file must also be updated to explain what those options actually do now.